task/net-api/src/lib.rs

```diff
@@ -50,6 +50,26 @@ pub enum RecvError {
     ServerRestarted = 2,
 }
 
+/// Errors that can occur when trying to set a VLAN as trusted
+#[derive(
+    Copy, Clone, Debug, PartialEq, Eq, FromPrimitive, IdolError, counters::Count,
```

Comment:
does this need a `Debug` impl? I don't see why it would unless it makes it to MGS?

Reply:
This is a case of preexisting condition copypasta – every other type in here also derives `Debug` 😉 

I removed it, although I don't believe it makes a difference since the compiler will strip it.

(There are a [_bunch_ of things](https://gist.github.com/mkeeter/ce77b889fd12cb26e4a10d2b5f16ca06) that derive debug; we could do a separate pass to clean them out)